### PR TITLE
use brfs transform on browserify

### PR DESCRIPTION
**Pull request by okdistribute**

### CODE REVIEW

**Comment by okdistribute** [comment]
otherwise hyperdrive gets broken in the browser!

File: package.json [hunk 1]
```diff
@@ -11,6 +11,14 @@
   "scripts": {
     "test": "tap test.js && standard"
   },
+  "browserify": {
+    "transform": [
+      "brfs"
+    ]
+  },
+  "devDependencies": {
+    "brfs": "^1.4.3"
```
Comment (juliangruber) [inline package.json]: can you move this to `dependencies`? Otherwise this won't work when another module requires this, as it won't have `brfs` installed.

**Comment by juliangruber** [comment]
otherwise LGTM!

File: package.json (hunk 1 above)
Comment (okdistribute) [reply package.json]: oh duh! OK :)

**Comment by okdistribute** [comment]
ok done!

**Comment by juliangruber** [comment]
rad! published as 2.0.1

**Pull request merged**